ui/selectors/multichain.ts

```diff
@@ -391,21 +422,17 @@ export function getMultichainIsTestnet(
   // the same pattern here too!
   const selectedAccount = account ?? getSelectedInternalAccount(state);
   const providerConfig = getMultichainProviderConfig(state, selectedAccount);
-  return getMultichainIsEvm(state, account)
-    ? // FIXME: There are multiple ways of checking for an EVM test network, but
```

Comment:
Can we re-add the `FIXME` and `TODO` here? They are still relevant IMO, and this logic is kinda fragile, so let's keep them.